tutorials/platform/android/android_library.rst

```diff
@@ -45,11 +45,11 @@ These APIs can also be used to provide bidirectional communication between the h
 Godot instance allowing for greater control over the desired experience.
 
 We showcase how this is done using a sample Android app that embeds the Godot Engine as an Android view,
-and uses it to render 3D GLTF models.
+and uses it to render 3D glTF models.
 
 The `GLTF Viewer <https://github.com/m4gr3d/Godot-Android-Samples/tree/master/apps/gltf_viewer>`_ sample app uses an `Android RecyclerView component <https://developer.android.com/develop/ui/views/layout/recyclerview>`_ to create
```

Comment:
I left the references to GLTF Viewer alone because it's the name of an external project.